libs/blocks/text/text.css

```diff
@@ -122,3 +176,17 @@
     padding-left: var(--spacing-l);
   }
 }
+
+/* Tablet */
+@media screen and (min-width:600px) and (max-width: 1200px) {
+  .link-farm.text-block .foreground {
```

Comment:
We should keep to the same viewport comments and declarations as in `typography.css` and shouldn't need to target an in-between query in most cases. 

```
/* Tablet up */
@media screen and (min-width: 600px) {}
/* Desktop up */
@media screen and (min-width: 1200px) {}
```
Since you're already calling the `Tablet Up` view port above, You could move this style into that media query, remove this one all together and it's already getting overwritten in the 'Desktop up' query below.

Reply:
Made the requested changes. Please review.